PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -305,22 +305,26 @@ public class AdUnit: NSObject, DispatcherDelegate {
     
     /// This method obtains the ext data keyword & value for adunit targeting.
     /// If the key already exists the value will be appended to the list. No duplicates will be added
+    @available(*, deprecated, message: "This method is deprecated. In the upcoming major release, the method will be removed. Please, use Targeting.addAppExtData method instead.")
```

Comment:
Are we replacing what goes into `imp[].ext.data`(`addExtData`) with what goes into `app.ext.data`(`Targeting.addAppExtData`)?